.gitignore

```diff
@@ -2,3 +2,114 @@
 __pycache__
 .mypy_cache/
 scan_build.egg-info/
+
```

Comment:
Is this relevant change? I would prefer to do without it.

Reply:
The build and dist and dist values appear in a git diff  along with some other files depending on the the environment you use. This prevents them from being possible to be added.

Which part of the change would you rather not have?

Reply:
I would just not add this change together the formatting one. :) Keep the PR focused on one thing at the time... You can submit another one with the `.gitignore` change if you think it's a good change.